instance/static/js/src/instance.js

```diff
@@ -57,48 +56,30 @@ app.factory('OpenCraftAPI', function(Restangular) {
 });
 
 
-// Function ///////////////////////////////////////////////////////////////////
-
-function updateInstanceList($scope, OpenCraftAPI) {
-    // Display loading message
-    $scope.loading = true;
-
-    OpenCraftAPI.all("openedxinstance").getList().then(function(instanceList) {
-        console.log('Updating instance list', instanceList);
-        $scope.instanceList = instanceList;
-
-        if($scope.selected.instance){
-            var updated_instance = null;
-            _.each(instanceList, function(instance) {
-                if(instance.id === $scope.selected.instance.id) {
-                    updated_instance = instance;
-                }
-            });
-            $scope.selected.instance = updated_instance;
-        }
-    }, function(response) {
-        console.log('Error from server: ', response);
-    }).finally(function () {
-        $scope.loading = false;
-    });
-}
-
-
 // Controllers ////////////////////////////////////////////////////////////////
 
 app.controller("Index", ['$scope', 'Restangular', 'OpenCraftAPI', '$q',
     function ($scope, Restangular, OpenCraftAPI, $q) {
-        // Display loading message
-        $scope.loading = true;
 
-        // Selection
-        $scope.selected = Array();
+        $scope.init = function() {
+            $scope.loading = true;
+            $scope.selected = Array();
```

Comment:
Is there a reason to use an array here?  It seems to be used like an object in the rest of the code, so `{}` would make more sense.